sycl/include/CL/sycl/backend/opencl.hpp

```diff
@@ -16,6 +16,7 @@
 #include <CL/sycl/detail/cl.h>
 #include <CL/sycl/kernel_bundle.hpp>
 
+#define SYCL_BACKEND_OPENCL 1
```

Comment:
Why is it defined apart from the others?

Reply:
others are extensions and this one is core part

Reply:
Is this file included by default?

Reply:
yes.